clusters/app.ci/ci-operator/registry-push-credentials-ci-central_secret_template.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+objects:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    annotations:
+      openshift.io/description: |
+        This secret contains a docker config which has the permissions to pull the images from local registry
+        and push the images to the ci central registry.
+
+    name: registry-push-credentials-ci-central
```

Comment:
Why is this `registry-push-credentials-ci-central` when the other PR uses `ci-central-registry-push-tokens` ?

Reply:
At the beginning i thought I do not need this secret.
I will update other PRs.

Reply:
`secret/ci-central-registry-push-tokens` provides the tokens to params defined in the templates of secrets here.
Those params will be given values based on the env. var. defined in the applyconfig.

`secret/registry-push-credentials-ci-central` generated from the templates here will be used in prowgen.
https://github.com/openshift/ci-tools/pull/1305

And then it is used by ci-operator in the promotion step which creates pods to do oc-image-mirror with docker's `config.json` from `secret/registry-push-credentials-ci-central`.

It contains the auth entires for pulling images from local registry and pushing images to the central registries: both app.ci's registry and api.ci's registry are there for easy migrating. We can remove the one for api.ci after the migration.

Reply:
I mean that the names should match, is all. If they are `ci-central-registry-push-tokens` then it should be `ci-central-registry-push-credentials`